pipes/WDL/workflows/tasks/tasks_read_utils.wdl

```diff
@@ -37,4 +37,32 @@ task downsample_bams {
   }
 }
 
+task dedup_bam {
+  File  in_bam
+  Int?  max_mismatches=5
```

Comment:
Seems kind of high?

Reply:
The [default for mvicuna](https://github.com/broadinstitute/mvicuna#running-vicuna) rmdup is 5 as well (`-drm_max_mismatch: default 5`). We can certainly reduce it.